src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -987,6 +987,22 @@ public function iShouldBeOnTheVariantGenerationPageForThisProduct(ProductInterfa
         Assert::true($this->variantGeneratePage->isOpen(['productId' => $product->getId()]));
     }
 
+    /**
+     * @Then I should see inventory of this product
+     */
+    public function iShouldSeeInventoryOfThisProduct(): void
+    {
+        Assert::true($this->updateSimpleProductPage->hasInventoryTab());
+    }
+
+    /**
+     * @Then I should not see inventory of this product
+     */
+    public function iShouldNotSeeInventoryOfThisProduct(): void
+    {
+        Assert::false($this->updateSimpleProductPage->hasInventoryTab());
```

Comment:
I guess it should be:
```suggestion
        Assert::false($this->updateConfigurableProductPage->hasInventoryTab());
```